examples/huggingface/run.py

```diff
@@ -103,13 +103,15 @@ def main(
     **kwargs
 ):
     if nlp_task == "token-classification":
-        # Run Pipeline
         token_classification_config = HuggingfaceParameters(
             label_all_tokens=True,
-            pretrained_model=pretrained_model,
-            epochs=epochs,
-            batch_size=batch_size,
+            pretrained_model=pretrained_model or "distilbert-base-uncased",
+            epochs=epochs or 1,
+            batch_size=batch_size or 16,
             dummy_run=not full_set,
+            dataset_name=kwargs.get("dataset_name") or "conll2003",
```

Comment:
I just checked the code and all of these already contain defaults from the click option? Wouldn't this `or` never be called?

Same for text_column and label_column